guides/common/attributes-base.adoc

```diff
@@ -72,13 +72,14 @@
 :oVirt: oVirt
 :oVirtEngine: {oVirt} Engine
 :oVirtShort: {oVirt}
-:package-clean: yum clean
+:package-clean: dnf clean
```

Comment:
The procedure I'm trying to fix uses both yum and dnf (from the same machine), and one of the things I'm trying to do is make it consistent. But instead of updating the attribute (thus changing :package-clean: elsewhere in the doc), hardcoding dnf clean in that particular procedure would probably work better. So I am going to revert this.